src/interfaces/project-create-interface.ts

```diff
@@ -15,6 +15,9 @@ export interface AppConfig {
   applicationName: string;
   appHostname: string;
   appPort: number;
+  isUpdate?: boolean;
```

Comment:
Having an `isUpdate` field in a file which is named `project-create-interface` seems like a big hack.